releasenotes/notes/[agent]-Add-deprecation-warnings-when-running-the-process-checks-on-the-process-agent-9a3eb8622e86e39e.yaml

```diff
@@ -0,0 +1,14 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Add deprecation warnings when running process checks on the process agent in linux.
+other:
+  - |
+    Change prepares for the deprecation of Process/Container Collection in the Process Agent, occurring in a future release.
```

Comment:
[minor] These actually go out into our [release notes](https://github.com/DataDog/datadog-agent/releases/tag/7.55.0). You can just combine them into a single section, `other` seems like a good spot, so the change makes more sense for an external reader.